eng/common/pipeline-logging-functions.ps1

```diff
@@ -101,7 +101,7 @@ function Write-PipelineTaskError {
       [string]$Path,
       [switch]$AsOutput)
 
-      if(-Not (Test-Path variable:ci) -Or !$ci) {
+      if((Test-Path variable:ci) -And $ci) {
```

Comment:
This change was made in arcade already, but had not propagated to arcade-services by the time I started working on it: 

https://github.com/dotnet/arcade/pull/4498/files

Reply:
When arcade-services tries to take a dependency update, this could conflict. Any chance this could be removed from this change so we don't have to worry about that?

Reply:
What was the resolution here?

Reply:
Reverted that change.

Reply:
Weird, codeflow was still showing the change as present, but GitHub is showing the right thing.

Reply:
@adiaaida without this change in here, my PR will not build. Arcade-services will need to have taken the arcade build with this change in it so I can rebase to it. If it's already in master, I'm not sure why I haven't gotten those changes when I rebased to master.

Reply:
The subscription is currently turned off since I didn't want to kill the cluster even more than it was before.

Reply:
I'm going to add this fix back in, then. We can deal with the merge conflict if it happens.

Reply:
I retriggered the subscription and the change is incoming in https://github.com/dotnet/arcade-services/pull/828. 

In the future, if you need to change a file in eng/common, you can get all the pending arcade changes locally to your branch by running
`darc update-dependencies --channel ".NET tools - latest" ` that way there's no chance of getting in an inconsistent state with what is in Arcade.

Reply:
subscription update PR has been committed.